components/infobox/wikis/starcraft2/infobox_league_custom.lua

```diff
@@ -95,14 +92,6 @@ function CustomInjector:parse(id, widgets)
 				content = {CustomLeague:_createPrizepool()},
 			},
 		}
-	elseif id == 'liquipediatier' then
-		return {
-			Cell{
-				name = 'Liquipedia tier',
-				content = {CustomLeague:_createLiquipediaTierDisplay()},
-				classes = {Logic.readBool(_args.featured) and 'tournament-highlighted-bg' or ''}
```

Comment:
Is highlighting not needed anymore?

Reply:
will get solved via the correct function overwrite^^